tests/runtest.cmd

```diff
@@ -90,7 +94,7 @@ if /i "%1" == "gcname"                (set COMPlus_GCName=%2&shift&shift&goto Ar
 if /i "%1" == "timeout"               (set __TestTimeout=%2&shift&shift&goto Arg_Loop)
 
 REM change it to COMPlus_GCStress when we stop using xunit harness
-if /i "%1" == "gcstresslevel"         (set __GCSTRESSLEVEL=%2&set __TestTimeout=1800000&shift&shift&goto Arg_Loop)
+if /i "%1" == "gcstresslevel"         (set COMPlus_GCStress=%2&set __TestTimeout=1800000&shift&shift&goto Arg_Loop)
```

Comment:
Are you going to remove the support for `__GCSTRESSLEVEL` from src\Common\Coreclr.TestWrapper\CoreclrTestWrapperLib.cs?

Reply:
Aren't we going to end up running GCStress on xunit? I guess xunit console runner is not managed code, so that's ok.

In general, I'd like to see us stop setting these JIT/GC stress variables here, and pass them as part of the "TestEnv" script.

Reply:
These environment variables are now picked up by runtest.py. A test env will be produced in python and passed to the test wrappers.

Reply:
Will runtest.py clear them from the environment? Or will they both be in the ambient environment *and* in the runtest.py-generated test env script?

Reply:
These will be unset by runtest.py here: https://github.com/dotnet/coreclr/pull/19213/files#diff-6c8dd7c2557f363fb660748401607f36R467.